src/contentScripts/index.ts

```diff
@@ -12,5 +13,12 @@ onMessage('tab-prev', ({ data }) => {
 
 // mount component to context window
 const container = document.createElement('div')
+const shadowDOM = container.attachShadow?.({ mode: 'closed' }) || container
```

Comment:
In consideration of debug experience, could we use `open` in dev ?

Reply:
updated.